Epsilon.UnitTest/Services/CompetenceDocumentServiceTests.cs

```diff
@@ -29,7 +29,7 @@ public async Task GetDocument_ReturnsCorrectDocumentWithFilteredDates()
         var userId = "testUser";
         var from = DateTime.Now.AddDays(-7);
         var to = DateTime.Now;
-        var expectedDocument = CompetenceDocumentService.FetchComponents(_submissions, new List<LearningDomain?>(){TestDataGenerator.GenerateRandomLearningDomain(), TestDataGenerator.GenerateRandomLearningDomain(),});
+        var expectedDocument = CompetenceDocumentService.FetchComponents(_submissions, new List<LearningDomain?>(){TestDataGenerator.GenerateRandomLearningDomain(), TestDataGenerator.GenerateRandomLearningDomain(),}, new List<LearningDomainOutcome>{TestDataGenerator.GenerateRandomLearningDomainOutcome(),});
```

Comment:
Now with testing, only 1 domain outcome is added. Please make it so that multiple domain outcomes are used with testing. 

In practice, around 40/50 domain outcomes will be used.